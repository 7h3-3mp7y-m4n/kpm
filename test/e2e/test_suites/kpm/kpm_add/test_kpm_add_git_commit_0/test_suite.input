kpm add -git https://github.com/kcl-lang/flask-demo-kcl-manifests.git -commit 0b3f5ab