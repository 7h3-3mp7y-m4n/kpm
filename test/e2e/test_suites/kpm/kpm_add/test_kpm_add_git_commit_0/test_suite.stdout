add dependency 'flask-demo-kcl-manifests:0b3f5ab' successfully